app/Services/OrderService.php

```
@@ -81,19 +91,57 @@ public function completeOrder(Order $order, array $paymentData = [])
+                'downloads_link' => route('user.downloads'),
+            ]);
+        } else {
+            sendNotification('ORDER_CONFIRMATION', $order->user, [
+                'user_name' => $order->user->name,
+                'order_code' => $order->code,
+                'order_total' => format_price($order->total),
+                'payment_method' => ucfirst(str_replace('_payment', ' ', $order->payment_method)),
+                'order_items_table' => $itemsTableHtml,
+                'downloads_link' => route('user.downloads'),
+            ]);
+        }
```

PHPMD (2.15.0)

The method completeOrder uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)